packages/core/src/App/Constants/routes-config.js

```diff
@@ -253,8 +253,8 @@ const getModules = ({ is_deriv_crypto }) => {
         },
         {
             path: routes.root,
-            component: Trader,
-            getTitle: () => localize('Trader'),
+            component: is_dashboard ? Dashboard : Trader,
+            getTitle: () => (is_dashboard ? localize('Dashboard') : localize('Trader')),
             routes: [
                 { path: routes.mt5, component: Trader, getTitle: () => localize('MT5'), is_authenticated: false },
                 {
```

Comment:
Should we edit these paths too? I'm not sure if we do want to have access to `Trader` component in dashboard if `is_dashboard` is true.

Reply:
@reza-fs just sorted this.